community/urls.py

```diff
@@ -8,6 +8,7 @@
 from django.views.generic import TemplateView
 
 from gci.views import index as gci_index
+from gci.view_twitter import index as twitter_index
```

Comment:
this doesnt compile.  `gci.view_twitter` doesnt exist

Reply:
you can see this on the travis build: https://travis-ci.org/coala/community/builds/316110663

Reply:
sorry, stupid mistake. Will fix it